python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -668,6 +715,58 @@ async def park_dome(self) -> None:
                 check_in_position=True,
             )
 
+    async def unpark_dome(self) -> None:
+        """Un-Park the dome by moving it a small delta amount."""
+
+        await self.assert_all_enabled(
+            message="All components need to be enabled for un-parking the Dome."
+        )
+
+        self.log.info("Checking if the dome is currently PARKED.")
+        az_motion = await self.rem.mtdome.evt_azMotion.aget(timeout=self.fast_timeout)
+
+        if az_motion.state == MTDome.MotionState.PARKED:
+            self.log.info("Dome is currently PARKED. Proceeding to un-park.")
+
+            current_position = await self.rem.mtdome.tel_azimuth.aget(
+                timeout=self.fast_timeout
+            )
+
+            az_delta = 0.1  # A small move to un-park the Dome
```

Comment:
Please make this a class attribute instead.

Reply:
shall we also add the
unpark_velocity = 0.0   # The Dome defaults to a crawl velocity

as an attribute?

Reply:
as per @tribeiro comment: 
`velocity will always be zero, no matter what.. you can even ignore that parameter and don't set it as the default is zero.`